### PR TITLE
fix: add support for legacy array definition method

### PR DESCRIPTION
**Pull request by Zxilly**
resolve: #308 

about: https://github.com/casbin/casbin/issues/863

Signed-off-by: Zxilly <zhouxinyu1001@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@lantu-group plz review

File: src/util/util.ts [hunk 1]
```diff
@@ -170,6 +170,31 @@ function deepCopy(obj: Array<any> | any): any {
   return newObj;
 }
 
+function bracketCompatible(exp: string): string {
+  // TODO: This function didn't support nested bracket.
+  if (!exp.includes(' (')) {
```
Comment (nodece) [inline src/util/util.ts:175]: please check `in `.

**Comment by nodece** [comment]
@Zxilly Please fix CI.

**Review by nodece — APPROVED** [review]
LGTM

**Comment by Zxilly** [comment]
@nodece I turned to a new implementation, plz review

**Pull request merged**